hack/tests/scorecard-subcommand.sh

```diff
@@ -25,5 +25,8 @@ echo $commandoutput | grep "Total Score: 80%"
 commandoutput2="$(operator-sdk scorecard \
   --proxy-image "$DEST_IMAGE" \
   --config "$CONFIG_PATH")"
-echo $commandoutput2 | grep "Total Score: 80%"
+# check basic suite
```

Comment:
We should add a test for JSON output and a test for human-readable format. These look like they are testing just one to me.